Makefile

```diff
@@ -448,8 +448,7 @@ build/toolchain/bin/protoc$(EXE_EXTENSION):
 	rm $(TOOLCHAIN_DIR)/protoc-temp.zip $(TOOLCHAIN_DIR)/readme.txt
 
 build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION):
-	$(GO) get github.com/golang/protobuf/protoc-gen-go
-	mv $(GO_BIN)/protoc-gen-go$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION)
```

Comment:
Missing the mkdir -p $(REPOSITORY_ROOT)/build/toolchain/bin/

`make clean build/toolchain/bin/protoc-gen-go` won't work otherwise.